pkgs/development/compilers/opensmalltalk-vm/vms.nix

```diff
@@ -0,0 +1,56 @@
+{
+  "linux64ARMv8" = {
+    "squeak.cog.spur" = {
+      configureFlagsArray = ''(
+        CFLAGS="-march=armv8-a -mtune=cortex-a72 -g -O2 -DNDEBUG -DDEBUGVM=0 -DMUSL -D_GNU_SOURCE -DUSEEVDEV -DCOGMTVM=0 -DDUAL_MAPPED_CODE_ZONE=1"
+        LIBS="-lrt"
+      )'';
+      configureFlags = [
+        "--with-vmversion=5.0"
+        "--with-src=src/spur64.cog"
+        "--without-npsqueak"
+        "--with-scriptname=spur64"
```

Comment:
I find this confusing, better to use `--with-scriptname=squeak` rather than `spur64`.